rocAL/rocAL_pybind/amd/rocal/pipeline.py

```diff
@@ -198,82 +195,82 @@ def __enter__(self):
     def __exit__(self, exception_type, exception_value, traceback):
         pass
 
-    def set_seed(self,seed=0):
+    def set_seed(self, seed=0):
         return b.setSeed(seed)
 
     @classmethod
-    def create_int_param(self,value=1):
-        return b.CreateIntParameter(value)
+    def create_int_param(self, value=1):
+        return b.createIntParameter(value)
 
     @classmethod
-    def create_float_param(self,value=1):
-        return b.CreateFloatParameter(value)
+    def create_float_param(self, value=1):
+        return b.createFloatParameter(value)
 
     @classmethod
-    def update_int_param(self,value=1,param=1):
-        b.UpdateIntParameter(value,param)
+    def update_int_param(self, value=1, param=1):
+        b.updateIntParameter(value, param)
 
     @classmethod
-    def update_float_param(self,value=1,param=1):
-        b.UpdateFloatParameter(value,param)
+    def update_float_param(self, value=1, param=1):
+        b.updateFloatParameter(value, param)
 
     @classmethod
-    def get_int_value(self,param):
-        return b.GetIntValue(param)
+    def get_int_value(self, param):
+        return b.getIntValue(param)
 
     @classmethod
-    def get_float_value(self,param):
-        return b.GetFloatValue(param)
-
-    def GetImageNameLen(self, array):
-        return b.getImageNameLen(self._handle, array)
+    def get_float_value(self, param):
+        return b.getFloatValue(param)
 
-    def GetImageName(self, array_len):
-        return b.getImageName(self._handle,array_len)
+    def get_image_name(self, array_len):
+        return b.getImageName(self._handle, array_len)
 
-    def GetImageId(self, array):
+    def get_image_id(self, array):
         b.getImageId(self._handle, array)
 
-    def GetBoundingBoxCount(self):
+    def get_bounding_box_count(self):
         return b.getBoundingBoxCount(self._handle)
 
-    def GetBoundingBoxLabels(self):
+    def get_bounding_box_abels(self):
```

Comment:
I thought we will be returning tensor lists for these APIs.

Reply:
This pipeline function calls the getBoundingBoxLabels function in rocal_pybind.cpp which returns the labels as lists.